examples/lang/flipper/src/lib.rs

```diff
@@ -42,6 +52,32 @@ contract! {
     impl Flipper {
         /// Flips the current state of our smart contract.
         pub(external) fn flip(&mut self) {
+            println(&format!("flip"));
+            let mut alloc = unsafe {
+                let mut fw_alloc = storage::alloc::BumpAlloc::from_raw_parts(Key([0x0; 32]));
+                let mut dyn_alloc = storage::alloc::DynAlloc::allocate_using(&mut fw_alloc);
+                dyn_alloc.initialize(());
+                dyn_alloc
+            };
+            let mut alloc = unsafe { BumpAlloc::from_raw_parts(Key([0x0; 32])) };
```

Comment:
Why are you using the `BumpAlloc` here instead of the `DynAlloc` in order to allocate the `storage::Vec` ?

Reply:
Accidental commit, was used for debugging.